### PR TITLE
update getting started documentation to be consistent with the rest of the documentation

### PR DESCRIPTION
**Pull request by sdadn**
### Summary

Closes https://jira2.cerner.com/browse/UXPLATFORM-5886

### Deployment Link

https://terra-graphs-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra
@cerner/carbon

### CODE REVIEW

File: packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/GettingStarted.a/GraphCore/Documentation/Graph.1.doc.mdx [hunk 1]
```diff
@@ -1,96 +1,151 @@
...
-            label: "Some Y2 Label",
-            lowerLimit: 0,
-            upperLimit: 250
-        }
+The graph configuration object has a set of properties for the underlying grid of a graph. 
+It hold various properties to configure a grid. 
+
+-   [Valid Graphs](#valid-graphs)
+-   [Configuration Structure](#configuration-structure)
+-   [Configuration Properties](#configuration-structure)
```
Comment (AshishMotanamGurunadham) [inline packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/GettingStarted.a/GraphCore/Documentation/Graph.1.doc.mdx:8]: change the hyperlink to configuration properties.
```suggestion
-   [Configuration Properties](#configuration-properties)
```

File: packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/GettingStarted.a/GraphCore/Documentation/Graph.1.doc.mdx [hunk 2]
```diff
@@ -1,96 +1,151 @@
...
 ```
+
+## Configuration Properties
+
+| Property Name          | Expected | Default                                        | Required | Description                                                                           |
+|------------------------|----------|------------------------------------------------|----------|---------------------------------------------------------------------------------------|
+| axis                   | object   | -                                              | yes      | See [Axis](./Axes).                                                                   |
+| bindTo                 | string   | -                                              | yes      | DOM id to bind the graph into.                                                        |
+| allowCalibration       | boolean  | `true`                                         | no       | Toggle to allow calibration to adjust the y axis.                                     |
+| bindLegendTo           | string   | `null`                                         | no       | If DOM id provided, binds legend into that container (Example: `"#legendContainer"`). |
```
Comment (AshishMotanamGurunadham) [inline packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/GettingStarted.a/GraphCore/Documentation/Graph.1.doc.mdx:74]: clickPassthrough link is not working as expected.

File: packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/GettingStarted.a/GraphCore/Documentation/Graph.1.doc.mdx [hunk 3]
```diff
@@ -1,96 +1,151 @@
...
+| showLabel              | boolean  | `true`                                         | no       | Toggle to show axes labels.                                                           |
+| showLegend             | boolean  | `true`                                         | no       | Toggle to show graph legend.                                                          |
+| showNoDataText         | boolean  | `true`                                         | no       | Toggle to show no data text.                                                          |
+| showShapes             | boolean  | `true`                                         | no       | Toggle to show shapes in the line graph.                                              |
+| showVGrid              | boolean  | `true`                                         | no       | Toggle to show vertical grid.                                                         |
+| ticksCount             | number   | `3`-`7` depending on the input range           | no       | See [ticksCount](#ticksCount).                                                        |
```
Comment (AshishMotanamGurunadham) [inline packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/GettingStarted.a/GraphCore/Documentation/Graph.1.doc.mdx:91]: tick Count hyperlink is working as expected.

File: packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/GettingStarted.a/GraphCore/Documentation/Graph.1.doc.mdx (hunk 2 above)
Comment (sdadn) [reply packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/GettingStarted.a/GraphCore/Documentation/Graph.1.doc.mdx:74]: fixed here: 741f50f900bbdf14d28172159fa3087416114641

File: packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/GettingStarted.a/GraphCore/Documentation/Graph.1.doc.mdx (hunk 3 above)
Comment (sdadn) [reply packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/GettingStarted.a/GraphCore/Documentation/Graph.1.doc.mdx]: fixed here: 741f50f900bbdf14d28172159fa3087416114641

**Review by AshishMotanamGurunadham — APPROVED** [review]

**Review by BenBoersma — APPROVED** [review]

**Review by tbiethman — APPROVED** [review]

**Pull request merged**